lib/ZumoOta/src/App.cpp

```diff
@@ -74,45 +83,66 @@ App::~App()
 {
 }
 
-void App::start() 
+bool App::loginit()
+{
+    if (true == Logging::getInstance().registerSink(&gLogSinkSerial))
+    {
+        (void)Logging::getInstance().selectSink("Serial");
+
+        /* Set severity of logging system. */
+        Logging::getInstance().setLogLevel(CONFIG_LOG_SEVERITY);
+
+        LOG_DEBUG("LOGGER READY");
+        return true;
+    }
+    else 
+    {
+        LOG_ERROR("Fail to init Logging!");
+        return false;
+    }
+
+} 
+
+void App::start()
 {
     if (m_fileManager.init())
     {
-        Serial.println("LittleFS initialization successful");
+        LOG_DEBUG("LittleFS initialization successful");
         m_webServer.init();
     }
     else
     {
-        Serial.println("LittleFS initialization failed. The application will not start.");
+        LOG_FATAL("LittleFS initialization failed. The application will not start.");
        
     }
 }
+  
 
 
 void App::setup()
 {
-  Serial.begin(SERIAL_BAUDRATE);
-// Access Point Modus start
-  /*WiFi.mode(WIFI_AP);
-  WiFi.softAP(ssid, password);
-
-  IPAddress IP = WiFi.softAPIP();
-  Serial.print("Access Point IP-Adresse: ");
-  Serial.println(IP);*/
-
-  //Station Mode start
-  WiFi.mode(WIFI_STA);
-  WiFi.begin(ssid, password);
-    if (WiFi.waitForConnectResult() != WL_CONNECTED) {
-        Serial.printf("WiFi Failed!\n");
-        return;
-    }
-
-    Serial.print("IP Address: ");
-    Serial.println(WiFi.localIP());
-
+    Serial.begin(SERIAL_BAUDRATE);

```

Comment:
shouln't this be part of logInit() or do you use Serial for anything else then logging ?

Reply:
It will be taken care of by the next commit.